crates/cli/src/parse/timestamp.rs

```diff
@@ -0,0 +1,47 @@
+use ethers::prelude::*;
+use polars::prelude::*;
+use cryo_freeze::{BlockChunk, Fetcher, ParseError, ChunkData, CollectError};
+
+async fn timestamp_to_block_number<P: JsonRpcClient>(
+    timestamp: u64,
+    fetcher: &Fetcher<P>
+) -> Result<u64, CollectError> {
+    let mut block_numbers: Vec<u64> = Vec::new();
```

Comment:
it looks like this variable doesnt do anything

Reply:
You're right, it's redundant. Will remove!